packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -215,13 +211,13 @@ servers:
       '/swagger-ui',
     );
     await server.get(RestBindings.PORT);
-    const url = new RegExp(
+    const urlString = new RegExp(
```

Comment:
Nitpick: the value is a regular expression, not a string. Please use `urlMatcher` or perhaps `expectedUrl` instead.